src/intTest/java/uk/nhs/digital/nhsconnect/nhais/container/MongoDbContainer.java

```diff
@@ -7,7 +7,7 @@
 public class MongoDbContainer extends GenericContainer<MongoDbContainer> {
 
     public static final int MONGODB_PORT = 27017;
-    public static final String DEFAULT_IMAGE_AND_TAG = "mongo:3.2.4";
+    public static final String DEFAULT_IMAGE_AND_TAG = "mongo:8.0";
```

Comment:
Thi was updated due to TestContainers reporting the following issue due docker versioning on the GitHub runners:

```
DEPRECATION NOTICE] Docker Image Format v1 and Docker Image manifest version 2, schema 1 support is disabled by default and will be removed in an upcoming release. Suggest the author of docker.io/library/mongo:3.2.4 to upgrade the image to the OCI Format or Docker Image manifest v2, schema 2`.
```